orchestra/scripts/add_slack_user_ids.py

```diff
@@ -0,0 +1,49 @@
+from annoying.functions import get_object_or_None
```

Comment:
would it be better as a migration actually that is skipped during testing? otherwise we need more setup docs about running this script etc